### PR TITLE
Attempt to use fedora 28 atomic host

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM if the tests are green.

**Comment by rhatdan** [comment]
Doubtful

**Comment by mheon** [comment]
Papr doesn't seem to be happy though

**Comment by rhatdan** [comment]
@cevich PTAL

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM

**Comment by giuseppe** [comment]
LGTM but it is unclear what the "internal error" is

**Comment by cevich** [comment]
> the "internal error" is

Just a guess:  Maybe the F28 image isn't available.  @jlebon would be the one to know for sure...or if something else "internal" broke.

**Comment by cevich** [comment]
...aye, it's those ``distro:`` values that matter the most.  Those must match something also known to papr, which is what I lack visibility into.

File: .papr.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ timeout: 45m
 tests:
```
Comment (jlebon) [inline .papr.yml]: You should be able to just drop the `packages:` block higher up since podman is already baked in FAH28.

File: .papr.yml [hunk 2]
```diff
@@ -45,18 +45,18 @@ context: "CAH smoketested"
...
+      baseurl: https://copr-be.cloud.fedoraproject.org/results/baude/Upstream_CRIO_Family/fedora-28-x86_64/
       gpgcheck: 0
 packages:
     - podman
     - buildah
 tests:
     - CONTAINER_RUNTIME="podman" sh .papr_prepare.sh
 required: false
-context: "Fedora fedora/27/cloud Podman"
+context: "Fedora fedora/28/cloud Podman"
```
Comment (jlebon) [inline .papr.yml]: Will check why this is failing.

**Comment by jlebon** [comment]
bot, retest this please

File: .papr.yml [hunk 3]
```diff
@@ -45,18 +45,18 @@ context: "CAH smoketested"
 
 inherit: true
 host:
-    distro: fedora/27/cloud/pungi
```
Comment (jlebon) [inline .papr.yml]: This is failing because there's no cloud pungi images. Try just `fedora/28/cloud` instead.

**Comment by rhatdan** [comment]
Looks like this is going to pass.
@mheon @wking @baude @umohnani8 @TomSweeneyRedHat PTAL

**Comment by mheon** [comment]
@rh-atomic-bot r+

**Comment by TomSweeneyRedHat** [comment]
FWIW, LGTM

**Pull request closed without merging**